index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse</title>
+</head>
+<body>
+    <h1>Hellow World</h1>

```

Comment:
- [ ] [optional] Kindly create the class for the `h1` tag and use classes to style the CSS because in a bigger project there may be many `h1` tags and it is not a good practice to style the project with an element